src/lib.rs

```diff
@@ -1622,21 +1610,28 @@ impl<'a, L: 'static + Ledger> KeystoreState<'a, L> {
                 let t = self
                     .txn_state
                     .add_pending_transaction(&mut model.transactions, &txn, info);
-                if let Err(err) = model.transactions.commit() {
+                if let Err(err) = model
```

Comment:
Now `transactions.commit()` is not getting called?

Reply:
it's called in `model.store()`

Reply:
Ah, good